Assets/Scripts/EnemyBody.cs

```diff
@@ -16,11 +16,15 @@ public class EnemyBody : MonoBehaviour
     protected Camera theCam;
 
     public Animator anim;
+
+    public AudioSource audioSource; // TODO: what will this be used for?
```

Comment:
Adding a reminder to add AudioSource components to enemies/logic for playing AudioClips